config/forecasters-co1e.yaml

```diff
@@ -29,6 +29,16 @@ analysis:
   label: COSMO KENDA
   analysis_zarr: /scratch/mch/fzanetta/data/anemoi/datasets/mch-co1e-an-archive-0p01-2019-2024-1h-v1-pl13.zarr
 
+stratification:
+  regions:
```

Comment:
we're also using the wildcard 'region' in the context of #50 (for the map region), should we change it there to something else? 

(I would use 'map' if that wouldn't be masking the built-in map function in python...)